config/config.yaml

```diff
@@ -41,8 +41,15 @@ storage:
     max_search_threads: 0
 
     # Max number of threads (jobs) for running optimizations across all collections, each thread runs one job.
+    # If 0 - have no limit and choose dynamically to saturate CPU.
     # Note: each optimization job will also use `max_indexing_threads` threads by itself for index building.
-    max_optimization_threads: 1
+    max_optimization_threads: 0
```

Comment:
`performance.max_optimization_threads: 0` now means there is no limit and the CPU should be saturated dynamically. This is the new default.

Even though we have a new `cpu_budget` parameter now, this still works. We can deprecate, but I guess it's fine to keep it as-is.